### PR TITLE
Notification: Number Badge on Main Tab

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1540
[Problem] Number Badge on MainTab is not implemented yet
[Solution]
  - MainTab widget modification
  - added examples in UIComponents
  - added new css styles
    
[Screenshot]
obraz
obraz

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: examples/mobile/UIComponents/components/notification-badge/number-badge-appbar.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+					</option>
+				</select>
+			</div>
+		</header>
+
+		<div class="ui-content">
+		</div>
+
+		<script>
+			var dropDownMenu = document.getElementById("sample-dropdown-menu");
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/notification-badge/number-badge-appbar.html]: move js file.

File: examples/mobile/UIComponents/components/notification-badge/number-badge-main-tab.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+				</li>
+				<li class="ui-li-anchor">
+					<a href="three.html">
+						Three
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="four.html">
+						Four
+						<span class="ui-badge" title="2"></span>
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/notification-badge/number-badge-main-tab.html:37]: what is the meaning of the tickle value?

File: src/css/profile/mobile/common/badge.less [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        position: relative;
+        padding: 0;
+        overflow: initial;
+        height: 17 * @px_base;
+        min-width: 17 * @px_base;
+        margin-left: 3 * @px_base;
+        align-self: center;
+
+        &::after {
+            top: -6 * @px_base;
```
Comment (DohyungLim) [inline src/css/profile/mobile/common/badge.less:62]: Do you process the value in minus to raise the badge a little?

**Review by DohyungLim — COMMENTED** [review]
3 comment.

File: src/css/profile/mobile/common/badge.less (hunk 3 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/badge.less:62]: This is according to guideline. The badge is positioning 6px above text which indicates.

File: examples/mobile/UIComponents/components/notification-badge/number-badge-appbar.html (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/notification-badge/number-badge-appbar.html]: This PR needs rebase.

File: examples/mobile/UIComponents/components/notification-badge/number-badge-main-tab.html (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/notification-badge/number-badge-main-tab.html:37]: The title attribute indicates badge content. If title not exists then the badge is only dot.

**Review by singa2000 — APPROVED** [review]
LGTM

**Pull request merged**